src/app/services/graph-client/msal-service.ts

```diff
@@ -82,8 +83,8 @@ export function logOutPopUp() {
   // @ts-ignore
   msalApplication.authorityInstance.resolveEndpointsAsync().then(authority => {
     const urlNavigate = authority.EndSessionEndpoint
-        ? authority.EndSessionEndpoint
-        : `${msalApplication.authority}oauth2/v2.0/logout`;
+      ? authority.EndSessionEndpoint
+      : `${msalApplication.authority}oauth2/v2.0/logout`;
```

Comment:
Does it mean that every time the version changes, this part of the code will be changed?
I think you could have the version (URL with version) defined somewhere as a constant such that if the version is updated later, a search need not be done to make updates but just a change in some one line. What do you think?